### PR TITLE
esther/GP 68/connect external website using LIFF

### PR DESCRIPTION
**Pull request by Esther125**
### Steps
- Follow the README.md to run the frontend. 
- Start your ngrok 
```
ngrok http 8081
```
- update _Endpoint URL_ with your ngrok url in LINE Developers settings.
(**NOTICE**: _Endpoint URL_ setting is in the User-Profile-LIFF Channel)
locate

### Sample Result
When you click on the menu, LIFF will open and an external website will appear. You can see the details with LIFF at the bottom of the website.

### CODE REVIEW

File: frontend/src/main copy.js
Comment (frafishen) [inline frontend/src/main copy.js:1]: why there's a file named `main copy.js` ?

**Comment by frafishen** [comment]
Pls grant me the admin permission for the `User-Profile-LIFF`!!!!

image

File: frontend/src/main copy.js
Comment (Esther125) [reply frontend/src/main copy.js:1]: I wanted to refer to the part written by 湯, so I copied it. But then I forgot to delete it lol. I've already removed it.

**Comment by yun-si** [comment]
I have coplete the steps above. But after I click the menu, 400 Bad Request shows.

**Review by yun-si — APPROVED** [review]
It works fine!

**Review by frafishen — APPROVED** [review]
LGTM!!!

**Pull request merged**